src/iso19111/factory.cpp

```diff
@@ -9171,7 +9171,8 @@ AuthorityFactory::createObjectsFromNameEx(
         auto sqlRes = d->run(sql, params);
         bool isFirst = true;
         bool firstIsDeprecated = false;
-        bool foundExactMatch = false;
+        size_t countExactMatch = 0;
+        size_t countExactMatchOnAlias = 0;
```

Comment:
Is this "alias" the alias from EPSG?

Reply:
> Is this "alias" the alias from EPSG?

yes, or ESRI. We request not only on the "name" field of objects, but also join the "alias_name" table